dashboard.py

```diff
@@ -28,14 +35,18 @@ def get_sample_info(dataset: BinsToVelocityDataset, midi_filename: str):
     return title, composer
 
 
-def model_predictions_review():
-    # options
-    path = st.selectbox(label="model", options=glob.glob("models/*.pt"))
-
-    start_index = eval(st.text_input(label="start index", value="0"))
+def model_predictions_review(cfg: DictConfig):
+    with st.sidebar:
+        # options
+        path = st.selectbox(label="model", options=glob.glob("models/*.pt"))
```

Comment:
I think it would be better to operate on run names rather than model files - that way we could reference the results in wandb easily. 

Also, looks like right now models are saved only for runs that actually finished, but I think it would be better to save every time there is an improvement in validation loss:

```python
best_test_loss = float("inf")  # or float(cfg.min_save_loss)
for epoch in epochs:
    train_epoch()
    v_loss = val_epoch():
    if v_loss <= best_test_loss:
        save_checkpoint(model, optimizer, cfg)
        best_test_loss = v_loss
```

If you then use those checkpoints here, you'll be able to show the full training config (maybe at the bottom).